cmd/generate/action/template/pkg/action/executor.go

```diff
@@ -54,10 +55,12 @@ func NewExecutor(config ExecutorConfig) (*Executor, error) {
 }
 
 func (e *Executor) Execute(ctx context.Context) error {
-	err := e.execute(ctx)
+	{{ if eq .Action "ac001" }}crs, err := e.execute(ctx){{ else }}err := e.execute(ctx){{ end }}
 	if err != nil {
 		return microerror.Mask(err)
-	}
+	}{{ if eq .Action "ac001" }}
+
+	config.SetCluster("{{ .Cluster }}", crs.Cluster.GetName()){{ end }}
```

Comment:
When executing the first action we usually create clusters. The used cluster ID is persisted for further action execution.